docs/markdown/Python-module.md

```diff
@@ -153,30 +153,54 @@ For example:
 purelib = py_installation.get_path('purelib')
 ```
 
-This function accepts a single argument, `path_name`, which is expected to
-be a non-empty string.
+This function requires at least one argument, `path_name`,
+which is expected to be a non-empty string.
+
+If `fallback` is specified, it will be returned if no path
+with the given name exists. Otherwise, attempting to read
+a non-existing path will cause a fatal error.
 
 **Returns**: A string
 
+### `has_path()`
+
+``` meson
+    bool py_installation.has_path(path_name)
+```
+
+**Returns**: true if a path of the given name exists and false otherwise.
+
 #### `get_variable()`
 
 ``` meson
-string py_installation.get_variable(variable_name)
+string py_installation.get_variable(variable_name, fallback)
 ```
 
 Get a variable as defined by the `sysconfig` module.
 
 For example:
 
 ``` meson
-py_bindir = py_installation.get_variable('BINDIR')
+py_bindir = py_installation.get_variable('BINDIR', '')
 ```
 
-This function accepts a single argument, `variable_name`, which is expected to
-be a non-empty string.
+This function requires at least one argument, `variable_name`,
+which is expected to be a non-empty string.
+
+If `fallback` is specified, it will be returned if no variable
+with the given name exists. Otherwise, attempting to read
+a non-existing variable will cause a fatal error.
 
 **Returns**: A string
 
+### `has_variable()`
+
+``` meson
+    bool py_installation.has_variable(variable_name)
+```
+
+**Returns**: true if a variable of the given name exists and false otherwise.
```

Comment:
This should specify that it is the same kind of `sysconfig` variable as for `get_variable`.